Rubberduck.VBEEditor/ComManagement/TypeLibs/TypeInfoConstantsCollection.cs

```diff
@@ -47,14 +47,31 @@ internal class TypeInfoConstantsCollection : TypeInfoVariablesCollection
 
                 // VBA constants are "static".... go figure. We can still infer it is a 
                 // constant rather than a field by checking the lpvarValue
-                if (varDesc.varkind == VARKIND.VAR_STATIC && varDesc.desc.lpvarValue != IntPtr.Zero)
+                if (varDesc.varkind == VARKIND.VAR_STATIC && IsValidPointer(varDesc.desc.lpvarValue))
                 {
                     _mapper.Add(_mapper.Count, i);
                 }
                 parent.ReleaseVarDesc(ppVarDesc);
             }
         }
 
+        /// <remarks>
+        /// On 64-bit platform, VBA seems to like putting random values in the high 32-bit part of the lpvarValue. Sometimes it does give out
+        /// null pointer but when ti does has no apparent reason or rhyme. Thus, to help avoid erroneous resolution of lpvarValue for vars that 
```

Comment:
Also @WaynePhillipsEA outlined in the issue description why the high word contains basically garbage, it might be useful to mention that in the documentation as well :)

Reply:
Agree. Will revise the paragraph. I'm still undecided if I'll make this an extension method so that it's available to other consumers (e.g. `ComModule`) which will have to deal with this issue that apparently exists only with VBA's faux `VAR_STATIC` stuff.